iml-agent/Cargo.toml

```diff
@@ -12,6 +12,7 @@ async-trait = "0.1"
 byte-unit = "=2.1"
 bytes = "0.5"
 chrono = "0.4"
+combine = "4.1.0"
```

Comment:
```suggestion
combine = "=4.1.0"
```

Reply:
Why do we want to fix the version like that?